dev/tests/integration/testsuite/Magento/GraphQl/_files/product_downloadable_without_purchased_separately_links.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+use Magento\TestFramework\Helper\Bootstrap as Bootstrap;
+use Magento\Catalog\Model\Product\Attribute\Source\Status as ProductStatus;
+use Magento\Downloadable\Model\Product\Type as ProductType;
+use Magento\Catalog\Model\Product\Visibility as ProductVisibility;
+
+/**
+ * @var \Magento\Catalog\Model\Product $product
+ */
+$product = Bootstrap::getObjectManager()->create(\Magento\Catalog\Model\Product::class);
+$product
+    ->setTypeId(ProductType::TYPE_DOWNLOADABLE)
+    ->setAttributeSetId(4)
+    ->setStoreId(1)
+    ->setWebsiteIds([1])
+    ->setName('GraphQl Downloadable Product (Links can not be purchased separately)')
```

Comment:
The description should not contain mentions about GraphQL, it's a general fixture in integration tests.

Reply:
**fixed**